Runtime/Native/INativeClient.cs

```diff
@@ -41,5 +41,10 @@ internal interface INativeClient : IDynamicAttributeProvider
         /// <param name="state">True if should pause, otherwise false.</param>
         void PauseAnrThread(bool state);
 
+        /// <summary>
+        /// Determine if the native integration layer is initialized
+        /// </summary>
+        /// <returns>True, if the native client is enabled</returns>
+        bool Initialized();

```

Comment:
small nit: shouldn't this be a getter?

```
bool Initialized { get; }
```

If not, maybe name the method `IsInitialized`?